spyder/plugins/console/widgets/console.py

```diff
@@ -151,7 +151,7 @@ class ConsoleBaseWidget(TextEditBaseWidget):
     # --- Signals
     # This signal emits an error text, which corresponds to a Python
     # traceback.
-    sig_exception_occurred = Signal(str, bool)
+    sig_exception_occurred = Signal(dict)
```

Comment:
Please complete this signal using a docstring and include the interface of the `dict`.
```python
sig_exception_occurred = Signal(dict)
"""
.......
"""
```

Please also follow https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Fixed